todo_app/templates/book_list.html

```diff
@@ -0,0 +1,16 @@
+<!doctype html> 
```

Comment:
This looks like it's left over from some of the other exercises too - it's probably worth removing this file now!